### ISSUE TITLE
Zoom does not work on Linux

### ISSUE THREAD

**Issue by rugk** [hakimel/reveal.js#2176]
On your example site https://revealjs.com/#/4 the zoom does not work. I do Alt+click, but it does not work in the demo site all.

Firefox 60

**Comment by Koriit** [comment]
@rugk, what is your OS? On Linux

**Comment by rugk** [comment]
Well…˝the slide said "Alt+click"… but yes, it works with Ctrl… thanks.

Just wonder why it does not use `Alt` there, too. It would totally be possible…

**Comment by Koriit** [comment]
According to @linux-man in #1909, ALT does not work on Linux.

**Comment by linux-man** [comment]
By defult, ALT+Click grabs the window from any spot of the window to easily move it around without having precisely aim for the titlebar like one would usually.

**Comment by linux-man** [comment]
I think it's the default behavior on KDE, Cinnamon, Mate, XFCE, Unity and most of the smaller DE. Gnome changed to SUPER+Click.

**Comment by linux-man** [comment]
You can change this behaviour on inicialize:

```
Reveal.initialize({
	// ...
	zoomKey: "alt",
	// ...
	width: "100%",
	height: "100%"
}); 
```
or after:

```
Reveal.configure({
	zoomKey: "alt"
});
```

**Comment by rugk** [comment]
Yeah, I'd rather worry about the standards. I mean, at least, it should be properly documented…

Or could not you just bind to both keys, so whatever one you can use on the platform works?

**Comment by hakimel** [comment]
fixed by @sophware in #2340
